code/modules/power/singularity/singularity.dm

```diff
@@ -75,6 +75,18 @@
 	consume(user)
 	return 1
 
+/obj/singularity/attack_tk(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		investigate_log("has consumed the brain of [key_name(C)] after being touched with telekinesis", "singulo")
+		C.visible_message("<span class='danger'>[C] suddenly slumps over.</span>", \
+		"<span class='userdanger'>As you mentally focus on the endless void you feel your very consciousness and soul get sucked into the singularity. That was an infinitely dense idea.</span>")
```

Comment:
"As you stare into the singularity, you feel your knowledge of the universe grow. So much raw information is there for the taking and you won't waste a speck of it. Within mere milliseconds you absorb as much information as possible, more than even the AI would have access to, you feel like a god. Suddenly, the gravity of this situation dawns on you as you feel your skull starting to collapse. That was a really dense idea, wasn't it?

Reply:
edited this a tad to fit more with being a tele interaction, but it was better than the one i had